db/load_fts_search.sql

```diff
@@ -29,4 +29,14 @@ INSERT INTO fts_search (result_id ,table_name ,LANGUAGE ,revision ,body ) SELECT
 INSERT INTO fts_search (result_id ,table_name ,LANGUAGE ,revision ,body ) SELECT c._id ,'CATEGORY' ,c.LANGUAGE ,c.revision ,s.body FROM CATEGORY C JOIN SECTION S ON S.CATEGORY_ID = C._ID WHERE C.LANGUAGE = 'uk' AND C.revision = "BA_2021";
 INSERT INTO fts_search (result_id ,table_name ,LANGUAGE ,revision ,body ) SELECT c._id ,'CATEGORY' ,c.LANGUAGE ,c.revision ,T.TAG FROM CATEGORY C JOIN TAGS T ON T.CATEGORY_ID = C._ID WHERE C.LANGUAGE = 'en' AND C.revision = "BA_2021";
 INSERT INTO fts_search (result_id ,table_name ,LANGUAGE ,revision ,body ) SELECT c._id ,'CATEGORY' ,c.LANGUAGE ,c.revision ,T.TAG FROM CATEGORY C JOIN TAGS T ON T.CATEGORY_ID = C._ID WHERE C.LANGUAGE = 'es' AND C.revision = "BA_2021";
-INSERT INTO fts_search (result_id ,table_name ,LANGUAGE ,revision ,body ) SELECT c._id ,'CATEGORY' ,c.LANGUAGE ,c.revision ,T.TAG FROM CATEGORY C JOIN TAGS T ON T.CATEGORY_ID = C._ID WHERE C.LANGUAGE = 'uk' AND C.revision = "BA_2021";
```

Comment:
Ukranian has it's own Cider 2015 version, we want to keep that until they can translate the new one.